app/templates/components/as-calendar.hbs

```diff
@@ -1,5 +1,8 @@
 {{#if showHeader}}
-  {{as-calendar/header title=title model=model}}
+  {{as-calendar/header 
+    title=title 
+    model=model 
+    onNavigateWeek=(action "onNavigateWeek")}}
```

Comment:
So, you agree I should leave it as is?

Let me know if there are any additional changes needed to get this merged in.